bin/htcondor_make_runtime.sh

```diff
@@ -10,11 +10,11 @@ STARTDIR=$PWD/tmp/runtime
 CRAB3_VERSION=3.3.0-pre1
 
 WMCOREDIR=$STARTDIR/WMCore
-WMCOREVER=1.1.14.crab1
+WMCOREVER=1.2.6
```

Comment:
is the version here really relevant ? I think it is overwritten later with what's in the spec file. Otherwside we'd be in troubles already.